### DISCUSSION TITLE
Vercel not redeploying latest commit to main

### DISCUSSION THREAD

**Discussion by mhendawi** [vercel/community#4623] [Help]
### Summary

Hi everything has been working great for several weeks.  I was affected by this issue for the last 24 hours:

https://github.com/vercel/remix/issues/62

I had to do an instant rollback to get my site back online.  Looks like issue is resolved and requires a redeploy.  Unfortunately Vercel is not redeploying.  I've tried five different commits now to get my project to redeploy but it's not picking up the changes for some reason.  What can I do to get it to redeploy my latest commits again.  It seems like something has broken.

This is for the project "jamspace".  

### Steps to Reproduce

The only steps are pushing a new commit.  It should trigger a redeploy as usual but is not after I did an instant rollback.

**Comment by amyegan** [comment]
Do you get any errors if you deploy with Vercel CLI?

https://vercel.com/docs/cli/deploying-from-cli#

**Reply by mhendawi** [reply]
Thanks for the response.  I transferred my project to a pro trial team but that didn’t change anything.  Then I tried to pay for a pro account so I could get support but it quoted me 40/month for just 1 account when i thought it would be 20.   So I transferred back to my personal hobby account and after that things started working again.  So I am not seeing any errors now from the CLI or deployment issues.  This happened to me about 5 weeks ago as well where I was unable to deploy reliably for about 24 hours. If it happens again I will try the CLI - I was not aware of that option.  I wish there was an option within the Vercel UI to just explicitly deploy a build based upon the latest commit.  The auto-detecting for deploys is really unreliable some days - other than not being able to deploy when I expect to, sometimes I’ve actually seen out of order deploys, where an older commit gets deployed after a more recent commit.
